apps/daffio/src/app/core/sidebar/containers/sidebar-viewport/sidebar-viewport.component.html

```diff
@@ -1,6 +1,9 @@
+<div></div>
 <daff-sidebar-viewport (backdropClicked)="close()" [mode]="mode$ | async" [opened]="showSidebar$ | async">
   <daff-sidebar class="daffio-sidebar" (escapePressed)="close()">
     <button type="button" daff-button class="daffio-sidebar__button">Get Started</button>
+    <h1>Guides</h1>
```

Comment:
This should be something other than an `h1` @xelaint

Reply:
I believe the sidebar component needs an overhaul? This would be handled in a separate PR.

Reply:
So I was the one who used the `h1` tag for this. The sidebar component could eventually be overhauled so I tried to make the `guide-viewer` component good to be reused if we do.

Reply:
@Nolan-Arnold I would just remove the `h1` for the time being.